### ISSUE TITLE
Add an expand/hide button on the lesson screen

### ISSUE THREAD

**Issue by BrandonArmand** [BrandonArmand/Binari#10]
- [ ] Bug
- [ ] Clean up
- [ ] Question
- [x] Enhancement
- [ ] Suggestion
- [ ] Other
 
## What needs to be done.
A button somewhere in the lesson screen to expand/hide the lesson quickly. It would probably fit best in the top right region.

<img width="702" alt="Screen Shot 2020-01-23 at 9 25 30 PM" src="https://user-images.githubusercontent.com/27185256/73046438-f8149800-3e26-11ea-9dc7-2ae862b01d79.png">

## Why
To quickly and seamlessly allow the user to work in the playground.

## How
It will probably be easy to add an icon with font-awesome and use CSS
```
.icon {
    position: absolute;
    top: ??;
    right: ??;
}
```
and implement JS/React to manipulate the height of the region. You can also find a better solution.

**Comment by alti21** [comment]
Hi can a try this?

**Comment by BrandonArmand** [comment]
@alti21 Absolutely!

**Comment by alti21** [comment]
@BrandonArmand I'm getting an error when running npm install for setting up the project

npm ERR! Unexpected end of JSON input while parsing near '....4","karma-sourcemap-'

**Comment by AdityaSharma911** [comment]
Hey! I would like to work on this issue. Is this still open?

**Comment by BrandonArmand** [comment]
@AdityaSharma911 It is (: I would love to check out what you build. It should be relatively simple to do, so I'm looking forward to seeing this issue finally close. 

Goodluck!

**Comment by AdityaSharma911** [comment]
Thanks! I will soon add a pull-request for the same.

**Comment by AdityaSharma911** [comment]
Sir, I was planning to insert the icon in the index.html and then style it in a new stylesheet but now I am getting confused about where to add the icon as the index.html in the public directory has no div components that the actual website renders. While inspecting the HTML of the actual page, I came across a different div tag then as shown in the index.html.  
Please help.
![help1](https://user-images.githubusercontent.com/55270982/89734557-698dfe00-da7a-11ea-92a3-197d24354818.JPG)

**Comment by BrandonArmand** [comment]
Hey @AdityaSharma911 

The HTML you see is stored in the JavaScript files under the Src folder. You will need to understand React.js, so if you don't already, I would recommend checking out their website and doing a quick course on how it works.

**Comment by guy-black** [comment]
Hi, this would be my first pull request so sorry if I did anything wrong.  I had a lot of trouble trying to pass the secondary panes height as a prop to Info, so I couldn't have it conditionally render an option to either shrink or expand depending on size like I wanted, so I added 3 icons to Info.js to expand, return to default size, and hide the lesson.  after pushing it to my fork I went ahead and opened a pull request.  I hope it looks good to you.

https://github.com/BrandonArmand/Binari/pull/57

![closed](https://user-images.githubusercontent.com/67166723/91075200-4b400a80-e603-11ea-9fc5-8a370a4b1383.PNG)
![expand](https://user-images.githubusercontent.com/67166723/91075201-4bd8a100-e603-11ea-85d1-ed96400912d0.PNG)
![open](https://user-images.githubusercontent.com/67166723/91075202-4bd8a100-e603-11ea-9f0e-8a8927e9486c.PNG)
